avm/res/network/express-route-circuit/tests/e2e/global-reach/dependencies.bicep

```diff
@@ -0,0 +1,90 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the first Express Route Port to create.')
+param expressRoutePort1 string
+
+@description('Required. The name of the second Express Route Port to create.')
+param expressRoutePort2 string
+
+@description('Required. The name of the first Express Route Circuit to create.')
+param expressRouteCircuit1 string
+
+// // Deploy first Express Route Port (for testDeploymentCircuit2)
+module testDeploymentPort1 'br/public:avm/res/network/express-route-port:0.3.1' = {
+  name: '${uniqueString(deployment().name, location)}-test-port1'
+  params: {
+    name: expressRoutePort1
+    location: location
+    bandwidthInGbps: 10
+    peeringLocation: 'Equinix-Amsterdam-AM5'
+    encapsulation: 'Dot1Q'
+    billingType: 'MeteredData'
+    tags: {
+      'hidden-title': 'Express Route Port for Circuit 1'
+      Environment: 'Non-Prod'
+      Role: 'DeploymentValidation'
+    }
+  }
+}
+
+// Deploy second Express Route Port (for later use)
+module testDeploymentPort2 'br/public:avm/res/network/express-route-port:0.3.1' = {
+  name: '${uniqueString(deployment().name, location)}-test-port2'
+  params: {
+    name: expressRoutePort2
+    location: location
+    bandwidthInGbps: 10
+    peeringLocation: 'Equinix-London-LD5'
+    encapsulation: 'Dot1Q'
+    billingType: 'MeteredData'
+    tags: {
+      'hidden-title': 'Express Route Port for Circuit 2Future Use'
+      Environment: 'Non-Prod'
+      Role: 'DeploymentValidation'
+    }
+  }
+}
+
+// Deploy second circuit
+module testDeploymentCircuit1 '../../../main.bicep' = {
```

Comment:
Dependencies must not use the module itself. Please either use a previously published version or the plain `resource`.
Dependencies should be 'stable' and not break upon changing anything in the module.

Reply:
Done.